src/pages/Beta/Swap/index.tsx

```diff
@@ -13,18 +13,18 @@ const SwapUI = () => {
   const chainId = useChainId()
   const useGelatoLimitOrders = useGelatoLimitOrdersHook[chainId]
   const { allOrders } = useGelatoLimitOrders()
-
+  const [swapType, onSwapTypeChange] = useState(SwapTypes.MARKET)
   const isLimitOrders = (allOrders || []).length > 0
   return (
     <PageWrapper>
       <TopContainer>
         <StatsWrapper>{isEvmChain(chainId) ? <PairInfo /> : <ComingSoon />}</StatsWrapper>
-        <SwapWidget isLimitOrderVisible={CHAINS[chainId]?.mainnet} />
+        <SwapWidget onSwapTypeChange={onSwapTypeChange} isLimitOrderVisible={CHAINS[chainId]?.mainnet} />
       </TopContainer>
 
       {CHAINS[chainId]?.mainnet && isEvmChain(chainId) && (
         <GridContainer isLimitOrders={isLimitOrders}>
-          {isLimitOrders && <LimitOrderList />}
+          {isLimitOrders && swapType == SwapTypes.LIMIT && <LimitOrderList />}
           <MyPortfolio />
           <WatchList coinChartVisible={!isLimitOrders} />
```

Comment:
`isLimitOrders={isLimitOrders}` & `coinChartVisible={!isLimitOrders}`

also needs to be updated as Grid & Watchlist depends on it

Reply:
I've fixed it